src/main/java/Main.java

```diff
@@ -0,0 +1,34 @@
+public class Main {
+
+    public static void main(String[] args) throws Exception {
+        BankApplication bankApplication = new BankApplication();
```

Comment:
In requirements of homework I described that you need to add separate method to process exceptions